openassessment/xblock/studio_mixin.py

```diff
@@ -232,7 +235,6 @@ def update_editor_context(self, data, suffix=''):  # pylint: disable=unused-argu
         Returns:
             dict with keys 'success' (bool) and 'msg' (str)
         """
-
```

Comment:
I believe this change is not related to the implementation, can we remove it?

Reply:
Sure